### DISCUSSION TITLE
关于paddleocr onnxruntime 部署问题

### DISCUSSION THREAD

**Discussion by ghost** [PaddlePaddle/PaddleOCR#12253] [Q&A]
您好，paddleocr 算法人员，关于paddleocr onnx 部署问题，paddleocr 提出了cpp_infer c++ 部署推理，请问这个是否支持onnx 模型，进行推理呢，目前想实现yolov8+ocr_v4模型进行onnx 部署，如果有新的主意或者消息请告诉我，感谢paddlocr算法人员

**Comment by DingHsun** [comment]
@SHOUshou0426 您好，我最近也在使用PaddleOCR，因為要用C++的onnxruntime推理部署，但礙於網路上找到的資源有限，因此在多方資源的拼湊之下我完成了PaddleOCR的部署，在我的網址：https://github.com/DingHsun/PaddleOCR-cpp 可以自行下載使用。

**Reply by QMCS-XC** [reply]
你好，请教一个问题，用C++推理onnx在使用GPU的情况下，显存占用正常，但是内存不断增长，换成CPU模式就不会出现这个问题，内存就会正常。这个问题有办法解决吗？

**Comment by ghost** [comment]
@DingHsun  谢谢您，同时也感谢您奉献出您的贡献，上面issues 我已经去实现完成了，并已经使用了，同时也非常感谢您

**Comment by DingHsun** [comment]
@SHOUshou0426 請問可否交流一下您是如何實現的呢？如果不方便也沒關係，感謝您。

**Comment by ghost** [comment]
@DingHsun  我查看了下您的实现，实现上面没有什么区别，模型预处理我使用的dnn 模块接口进行的处理，同时我前面检测算法，修改成了yolov8，将yolov8和ocr rec 模型进行了融合

**Comment by Li-Yidong** [comment]
> @DingHsun 我查看了下您的实现，实现上面没有什么区别，模型预处理我使用的dnn 模块接口进行的处理，同时我前面检测算法，修改成了yolov8，将yolov8和ocr rec 模型进行了融合

您好，我最近也想使用opencv的dnn模块来部署paddleocr，请问您使用的opencv是哪个版本的？

**Comment by DingHsun** [comment]
@Li-Yidong 
我實現所使用的packages版本在 https://github.com/DingHsun/PaddleOCR-cpp 裡的Packages，opencv使用4.5.0。

**Comment by Li-Yidong** [comment]
@DingHsun 感谢您的分享
